scripts/build_utils.sh

```diff
@@ -600,10 +603,13 @@ setup_pbuilder() {
         echo "$other_mirror" >> ~/.pbuilderrc
     fi
 
+    #in case it needs to update/add repo, the following packages should installed.
+    packages_for_repo_update="software-properties-common curl"
+
     if [ $FLAVOR = "crimson" ]; then
-        extrapackages='EXTRAPACKAGES="libc-ares-dev libcrypto++-dev libgnutls28-dev libhwloc-dev libnuma-dev libpciaccess-dev libprotobuf-dev libsctp-dev libyaml-cpp-dev protobuf-compiler ragel systemtap-sdt-dev"'
-        echo "$extrapackages" >> ~/.pbuilderrc
+        extrapackages="EXTRAPACKAGES=\"libc-ares-dev libcrypto++-dev libgnutls28-dev libhwloc-dev libnuma-dev libpciaccess-dev libprotobuf-dev libsctp-dev libyaml-cpp-dev protobuf-compiler ragel systemtap-sdt-dev ${packages_for_repo_update}\""
     fi
+    echo "$extrapackages" >> ~/.pbuilderrc
```

Comment:
```bash
extrapackages="software-properties-common curl"
if [ $FLAVOR = "crimson" ]; then
  extrapackages += " software-properties-common curl"
fi
echo "EXTRAPACKAGES=${extrapackages}" >> ~/.pbuilderrc
```

as discussed offline. i think we should enable "software-properties-common curl" for the flavors which are not "crimson". while your change does not take this into consideration.